app/src/lib/stores/app-store.ts

```diff
@@ -1535,6 +1535,16 @@ export class AppStore {
     this.emitUpdate()
   }
 
+  /** This shouldn't be called directly. See `Dispatcher`. */
+  public async _closeCurrentFoldout(): Promise<void> {
+    if (!this.currentFoldout) {
```

Comment:
Can you use an explicit check here? Using a `!` is a bit too overloaded, since we could be checking for `null`, `undefined`, 0, `''`, etc.

Reply:
Addressed in 08a435d9b439193d70663afb591c42ef145100e8. 🕵️